ssh_automated_tests/core/property_reader.py

```diff
@@ -0,0 +1,18 @@
+import configparser
+from abc import abstractmethod, ABC
+
+
+class Reader(ABC):
+    @abstractmethod
+    def read(self, parameter: str):
+        pass
+
+
+class ConfigReader(Reader):
+    def __init__(self, file):
+        self._config = configparser.ConfigParser()
+        self._source_file = file
+
+    def read(self, parameter: str):
```

Comment:
It sounds a bit crazy that the `read` method returns something. May we can identify a better name for this method?